source/language/types.rst

```diff
@@ -268,6 +268,72 @@ are all of type ``float[64]``.
+      +----------+--------------------------------+---------------------+
+      | ceiling  | [float], [float]               | [float]             |
+      +----------+--------------------------------+---------------------+
+      | cos      | [float]                        | [float]             |
+      |          |                                |                     |
+      |          | [angle]                        | [float]             |
+      +----------+--------------------------------+---------------------+
+      | div      | ??                             | ??                  |
+      +----------+--------------------------------+---------------------+
+      | exp      | [float]                        | [float]             |
+      |          |                                |                     |
+      |          | [complex]                      | [complex]           |
+      +----------+--------------------------------+---------------------+
+      | floor    | [float], [float]               | [float]             |
+      +----------+--------------------------------+---------------------+
+      | log *(e)*| [float]                        | [float]             |
+      +----------+--------------------------------+---------------------+
+      |  mod     | [int], [int]                   | [int]               |
+      |          |                                |                     |
+      |          | [float], [int,float]           | [float]             |
+      +----------+--------------------------------+---------------------+
+      | popcount | [bits, unsigned]               | [unsigned int]      |
+      +----------+--------------------------------+---------------------+
+      | pow      | [int], [unsigned int]          | [int]               |
+      |          |                                |                     |
+      |          | [float], [int,float]           | [float]             |
+      |          |                                |                     |
+      |          | [complex], [int,float,complex] | [complex]           |
+      +----------+--------------------------------+---------------------+
+      | rotl     | [bit], [int,uint]              | [int,uint]          |
+      +----------+--------------------------------+---------------------+
+      | rotr     | [bit], [int,uint]              | [int,uint]          |
+      +----------+--------------------------------+---------------------+
+      | sin      | [float]                        | [float]             |
+      |          |                                |                     |
+      |          | [angle]                        | [float]             |
+      +----------+--------------------------------+---------------------+
+      | sqrt     | [float] *principal root*       | [float]             |
+      |          |                                |                     |
+      |          | [complex] *principal root*     | [complex]           |
```

Comment:
A "notes" column would be a good place to put the "principal root" comments in as well.